### PR TITLE
owners: updating codeowners file

### PR DESCRIPTION
**Pull request by alyssawilk**
Adding codeowners for unowned extension directories, mostly based on git blame, and sometimes based on maintainer suggestions.  

As a reminder, codeowners are the people we'll generally default to for reviews of files in a given directory.  An Envoy maintainer still has to LGTM in order to merge, and generally that'd end up being the maintainer listed in codeowners.

Risk Level: n/a
Testing: n/a
Docs Changes: n/a
Release Notes: n/a

### CODE REVIEW

**Comment by alyssawilk** [comment]
@snowp @alyssawilk @eziskind @htuch @mattklein123 @zuercher @gsagula @dio @jose @rshriram @qiwzhang @lizan @fengli79 @lavignes @yuval-k @klarose @PiotrSikora @wozz @taiki45 @trabetti @ramaraochavali

File: CODEOWNERS [hunk 1]
```diff
@@ -59,3 +59,31 @@ extensions/filters/common/original_src @snowp @klarose
...
 /*/extensions/common/crypto @lizan @PiotrSikora @bdecoste
+/*/extensions/filters/http/grpc_http1_bridge @snowp @jose
+/*/extensions/filters/http/gzip @gsagula @dio
+/*/extensions/filters/http/fault @rshriram @alyssawilk
+/*/extensions/filters/common/fault @rshriram @alyssawilk
+/*/extensions/filters/http/grpc_json_transcoder @qiwzhang @lizan
+/*/extensions/filters/http/router @alyssawilk @mattklein123 @snowp
+/*/extensions/filters/http/ext_authz @gsagula @dio
+/*/extensions/filters/http/grpc_web @fengli79 @lizan
+/*/extensions/filters/http/common/aws @lavignes @dio
```
Comment (lavignes) [inline CODEOWNERS]: I did cover this on line 51

Comment (lavignes) [reply CODEOWNERS]: Just wanted to call it out especially since the two lines didn't match :)

Comment (alyssawilk) [reply CODEOWNERS]: oh good catch, thanks!  Must have been after we landed the fix format checks and we just forgot to take it out.

**Pull request merged**